crates/serpen/tests/fixtures/ast_rewriting/class_name_collision/main.py

```
@@ -0,0 +1,29 @@
+    model_user = ModelUser("Alice", "alice@example.com")
+    entity_user = EntityUser("Bob", 25)
+
+    model_product = ModelProduct("Widget", 19.99)
+    entity_product = EntityProduct("P001", "Gadget")
+
+    print(f"Model User: {model_user}")
+    print(f"Entity User: {entity_user}")
+    print(f"Model Product: {model_product}")
+    print(f"Entity Product: {entity_product}")
+
+    return {"model_user": str(model_user), "entity_user": str(entity_user), "model_product": str(model_product), "entity_product": str(entity_product)}
```

Pylint (3.3.7)

[convention] Line too long (151/100)

(C0301)